markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+---
+title: "Metadata Template"
+author: "Christian Krestel"
+date: "2023-06-27"
+output:
+  pdf_document:
+    includes:
+      in_header: "preamble.tex"
+    toc: true
+    toc_depth: 3
+    latex_engine: xelatex
+---
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Version number: 3
+
+# Template: Code repository catalogue
+
+1. Create new project: Type script
+2. Screenshot of the \emph{complete} first page
+3. For a published paper, prepare the link to the repository. For a WP prepare a zip that contains all data and code; For \textbf{DIME verified} packages also the reproducibility verification statement
+
+# Information on metadata
+This is information about the creator of the metadata. Sufficient fields
+\begin{enumerate}
+  \item Name: Second name only: Smith
+  \item Abbreviation: PS
+  \item Name: Smith
+  \item Affiliation: World Bank
+  \item Production date: Year-Month-Day
+  \item Version number: Entered as 1,2,3
```

Comment:
clarify whether this is metadata version or reproducibility package version.

Reply:
For meta data

Reply:
cool, that should be noted in the protocol